electron/Transport.js

```diff
@@ -0,0 +1,16 @@
+const Transport = require('winston-transport');
+const stripAnsi = require('strip-ansi');
+
+module.exports = class CustomTransport extends Transport {
+  constructor(opts) {
+    super(opts);
+    this.loggedMessages = opts.loggedMessages;
+  }
+
+  log(info, callback) {
+    const level = stripAnsi(info.level);
+    this.loggedMessages.push(`${info.timestamp} [${level}]: ${info.message}`);
```

Comment:
DP3 - Now that we have the stripAnsi function, do we need to handmake this message anymore? Is there some way of getting this fully formatted log message off of the `info` object, instead of recreating the format ourselves?